test/Unit/Api/ClientTest.php

```diff
@@ -322,6 +323,11 @@ public function testGettingTheCountOfACollectionReturnsThatCollectionsCount($pat
             return;
         }
 
+        //This isn't really something needed ATM for this group of functions
+        if ($path == 'pages'){
```

Comment:
Oh duh. For some reason I was not reading that as `in_array` but some imaginary function that was checking for all params to exist in string.